models/base_model.py

```diff
@@ -32,8 +32,7 @@ def create(cls, schema, payload):
             abort(400, err.messages)
 
         obj = cls(**attrs)
-        db.session.add(obj)
```

Comment:
Nice work drying this up here~

Reply:
This change was actually necessary which highlights the fact that I should have a test for this.

Reply:
I will address in this ticket https://github.com/codeforpdx/dwellingly-app/issues/426